packages/flutter/lib/src/rendering/editable_line.dart

```diff
@@ -147,6 +153,31 @@ class RenderEditableLine extends RenderBox {
 
   bool hitTestSelf(Point position) => true;
 
+  TapGestureRecognizer _tap;
+  void handleEvent(PointerEvent event, BoxHitTestEntry entry) {
+    if (event is PointerDownEvent && onSelectionChanged != null)
+      _tap.addPointer(event);
+  }
+
+  Point _lastTapDownPosition;
+  void _handleTapDown(Point globalPosition) {
+    _lastTapDownPosition = globalPosition;
+  }
+
+  void _handleTap() {
+    assert(_lastTapDownPosition != null);
+    final Point global = _lastTapDownPosition;
+    _lastTapDownPosition = null;
+    if (onSelectionChanged != null) {
+      TextPosition position = _textPainter.getPositionForOffset(globalToLocal(global).toOffset());
+      onSelectionChanged(new TextSelection.fromPosition(position));
```

Comment:
(This is a bit weird. The RenderObject model would be to have the RenderObject own and mutate its own selection rather than having the owner be in charge of how to respond to selection move requests.)